README.md

```diff
@@ -36,21 +36,21 @@ To set it up, follow the steps [here](https://github.com/microsoft/vscode-jupyte
 
 ## Working with other Languages
 
-The Jupyter Extension supports other languages in addition to Python such as Julia, R, and C# in VS Code Insiders with our latest Native VS Code Notebooks Experience!
+The Jupyter Extension supports other languages in addition to Python such as Julia, R, and C#.
 
 ### Quick Start
 
--   **Step 1.** Install [VS Code Insiders](https://code.visualstudio.com/insiders/)
+-   **Step 1.** Install [VS Code](https://code.visualstudio.com/)
 
 -   **Step 2** If not working with Python, make sure to have a Jupyter kernelspec that corresponds to the language you would like to use installed on your machine.
 
 -   **Step 3.** Install the [Jupyter Extension](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter)
 
--   **Step 4.** Open or create a notebook file and start coding!
+-   **Step 4.** Open or create a notebook file and start coding.
```

Comment:
I removed basically all !s out of a strong personal preference ;). However I'm totally willing to put them back in if I'm outvoted here.